sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -1471,6 +1523,12 @@ public partial class IisLogsDataSource
         public string Name { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> Streams { get { throw null; } }
     }
+    public partial class IngestionSettings
+    {
+        internal IngestionSettings() { }
+        public string DataCollectionEndpointResourceId { get { throw null; } }
+        public string DataCollectionRuleResourceId { get { throw null; } }
```

Comment:
Are these two arm resource id? If so please change their format to `arm-id`

Reply:
where should we change it to arm-id?

Reply:
Added -|arm-id formatting, types updated to `Azure.Core.ResourceIdentifier`